README.md

```diff
@@ -202,6 +202,39 @@ PlaybackSDKManager.shared.initialize(
 
 By default the SDK uses system user agent, so if your app uses native URL Session, the `userAgent` parameter most likely can be omitted.
 
+## Bitmovin analytics
+
+Currently SDK support tracking analytics on Bitmovin service. In case you have a logged-in user and want to track Bitmovin analytics for the current session, you need to pass the user's ID in the `analyticsViewerId` parameter.
+
+Example: 
+
+```swift
+    let entryId = "..."
+    let authorizationToken = "..."
+    let analyticsViewerId = "user id or empty string"
+    
+    /// ** Load player with the playback SDK **
+    PlayBackSDKManager.shared.loadPlayer(entryID: entryId,
+                                            authorizationToken: authorizationToken,
+                                            mediaTitle: "Background audio test",
```

Comment:
@KharchenkoAlex can you check the `mediaTitle` parameter? I think with the new version that param is no longer there. Other than that, what do you think to document the `loadPlaylist` also? It should possible to pass the `analyticsViewerId` there also.

Reply:
@StefanoStream updated